.github/workflows/solidity.yml

```diff
@@ -6,7 +6,11 @@ on:
       - auto
       - try
       - rollup
+    paths:
+      - 'contracts/**'
```

Comment:
Another path to add is the root yarn.lock, otherwise we might miss dependency updates that are still valid within semver constraints, but cause breaking changes within our tests.